guide/src/setting_up_your_development_environment.md

```diff
@@ -135,13 +135,19 @@ Note: The source of the Cloud Client Libraries for Rust is
    cargo run [PROJECT ID]
    ```
 
+   The program will print the secrets associated with your project ID. If you
+   don't see any secrets, you might not have any in Secret Manager. You can
+   [create a secret] and rerun the program, and you should see the secret 
```

Comment:
nit: looks like this line ends in a space. the linter is unhappy about it.

Reply:
Fixed!